### PR TITLE
News feed

### LINKED ISSUE #1407: Improve the news section on the main page.
It would be nice if the news section has a vertical scrolling feature so that it becomes easy to add more news feeds later on without wasting much space. Also, it will be easy to shift this section whenever needed.

image

### PR DESCRIPTION
**Pull request by guptadiksha307**
# Issue Description

Added scrolling functionality in the news section

Fixes #1407

## Changes Made

It would be nice if the news section has a vertical scrolling feature so that it becomes easy to add more news feeds later on without wasting much space. Also, it will be easy to shift this section whenever needed.

image

### CODE REVIEW

File: site/learn/index.md [hunk 1]
```diff
@@ -77,25 +77,21 @@
           Learn more, and <a href="https://www.fun-mooc.fr/courses/course-v1:parisdiderot+56002+session03/about">register now on the FUN platform!</a>
 	  </p>
 		  <p class="documentation-video" style="margin-bottom:0">
-<iframe title="Xavier Leroy presents the state of OCaml 4.02" src="//www.slideshare.net/slideshow/embed_code/43836300" width="340" height="290" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px; margin-bottom:5px; max-width: 100%;" allowfullscreen> </iframe>
+<iframe title="Xavier Leroy presents the state of OCaml 4.02" src="//www.slideshare.net/slideshow/embed_code/43836300" width="380" height="220" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px; margin-bottom:5px; max-width: 100%;" allowfullscreen> </iframe>
           </p>
 
-          <p>An invited talk by Xavier Leroy explaining the current state
-		  of OCaml at the
+<p>An invited talk by Xavier Leroy explaining the current state of OCaml at the
```
Comment (pitag-ha) [inline site/learn/index.md:82]: I suppose this formatting was changed by mistake? If so, please correct this. It's something typical that can happen, so don't worry. I won't point out every formatting change, but please also correct the others.

**Review by pitag-ha — COMMENTED** [review]
As I've pointed out on the issue, I'm not quite convinced yet if this ends up being an improvement. But I'd love to hear your motivation for this change; you might convince me :) The right place for that is the issue. I've already asked a couple of questions there.

Some general feedback regardless of weather this gets merged: 
- It's important to link the corresponding issue correctly. In this case that means writing "Fixes #1407" instead of "Fixes # (1407)" (you can edit that)
- Avoid formatting changes (I've pointed out one of them)
- Maintain a clean git history. In most cases, including this one, adding one commit per PR is enough. Several of the commits you have are merge commits; those should be avoided for sure. But it's not a problem. You can fix that via `git rebase -i <sha_of_first_commit_added>` using the `squash` option; here concretely `git rebase -i 2a9a2756c0ac299d059c487c13c09c86684b88c1`.

But apart from that the PR and the PR description are perfectly fine :) We just need to discuss the content (on the issue).

**Comment by avsm** [comment]
This PR seems to conflict with the main branch of the repository. If you could resolve the conflicts and/or remove unnecessary changes to the PR, that would help review it. Thanks!

**Comment by guptadiksha307** [comment]
> This PR seems to conflict with the main branch of the repository. If you could resolve the conflicts and/or remove unnecessary changes to the PR, that would help review it. Thanks!

Yes, I'm working on it :)

**Comment by pitag-ha** [comment]
On a phone, a scroll bar in the news section impacts the scroll experience through the page (see  [this screen recording](https://github.com/ocaml/ocaml.org/issues/1407#issuecomment-816225492)). That's not a strict objection from my side, but I wanted to point that out :)

**Comment by avsm** [comment]
Thanks for this contribution, and for the associated discussion in #1407. I've reviewed it on the various setups (main screen, mobile) and do not think that it's an improvement over the current site. The reason for this is that it makes use on mobile trickier (due to the scrolling issue pointed out by @pitag-ha) and on the main site we have no plans to increase the number of news items on the front page (there is a link to the wider planet site).

It's important to note that we couldn't have to come to this view without this PR being created in order to experiment with the scrollbar, so this is PR is a very positive contribution that I consider a "success". It's important to keep experimenting and trying out different approaches.  Please do keep them coming @guptadiksha307 :-)

**Pull request closed without merging**